### PR TITLE
 Removes remaining I18n props, HOCs and misc objects

### PR DESCRIPTION
**Pull request by AlexSCorey**
##### SUMMARY
This Removes `withI18n`, the `i18n` object and all the places that it is passed around as a prop, or argument.  

~E2E Tests have been triggered.  There should be no functional or visual impact~. [E2E results](https://dashboard.cypress.io/projects/ui2mq7/runs?branches=%5B%7B%22label%22%3A%22RemoveI18nStuff%22%2C%22suggested%22%3Afalse%2C%22value%22%3A%22RemoveI18nStuff%22%7D%5D&committers=%5B%5D&flaky=%5B%5D&page=1&status=%5B%5D&tags=%5B%5D&timeRange=%7B%22startDate%22%3A%221970-01-01%22%2C%22endDate%22%3A%222038-01-19%22%7D)

##### ISSUE TYPE
-Dependency Upgrade

##### COMPONENT NAME
 - UI

##### AWX VERSION

```

```

### CODE REVIEW

File: awx/ui_next/src/components/AlertModal/AlertModal.jsx [hunk 1]
```diff
@@ -20,13 +20,12 @@ const Header = styled.div`
...
 
 function AlertModal({
-  i18n,
   isOpen = null,
   title,
   label,
   variant,
   children,
-  i18nHash,
+  Hash,
```
Comment (nixocio) [inline awx/ui_next/src/components/AlertModal/AlertModal.jsx]: This prop is not used on this component.

File: awx/ui_next/src/components/ChipGroup/ChipGroup.jsx [hunk 2]
```diff
@@ -1,24 +1,23 @@
 import React from 'react';
-import { withI18n } from '@lingui/react';
+
 import { t } from '@lingui/macro';
 import { ChipGroup as PFChipGroup } from '@patternfly/react-core';
-import { number, shape } from 'prop-types';
+import { number } from 'prop-types';
 
-function ChipGroup({ i18n, numChips, totalChips, i18nHash, ...props }) {
+function ChipGroup({ numChips, totalChips, Hash, ...props }) {
```
Comment (nixocio) [inline awx/ui_next/src/components/ChipGroup/ChipGroup.jsx]:
```suggestion
function ChipGroup({ numChips, totalChips, ...props }) {
```

File: awx/ui_next/src/components/CredentialChip/CredentialChip.jsx [hunk 3]
```diff
@@ -1,15 +1,14 @@
...
-import { shape } from 'prop-types';
-import { withI18n } from '@lingui/react';
+
 import { t } from '@lingui/macro';
 import { Chip } from '@patternfly/react-core';
 import { Credential } from '../../types';
 import { toTitleCase } from '../../util/strings';
 
-function CredentialChip({ credential, i18n, i18nHash, ...props }) {
+function CredentialChip({ credential, Hash, ...props }) {
```
Comment (nixocio) [inline awx/ui_next/src/components/CredentialChip/CredentialChip.jsx]:
```suggestion
function CredentialChip({ credential, ...props }) {
```

**Review by nixocio — APPROVED** [review]
This looks great @AlexSCorey. 

I left minor suggestions.

We should update the contributing guide removing the HOC - on the session.
> Marking strings for translation and replacement in the UI

File: awx/ui_next/CONTRIBUTING.md [hunk 4]
```diff
@@ -336,16 +336,12 @@ Internationalization leans on the [lingui](https://github.com/lingui/js-lingui)
...
 
-- import the withI18n function and wrap the export of your component in it (i.e. `export default withI18n()(Foo)`)
-- doing the above gives you access to the i18n object on props. Make sure to put it in the scope of the function that contains strings needed to be translated (i.e. `const { i18n } = this.props;`)
 - import the t template tag function from the @lingui/macro package.
-- wrap your string using the following format: `` i18n._(t`String to be translated`) ``
-
-**Note:** Variables that are put inside the t-marked template tag will not be translated. If you have a variable string with text that needs translating, you must wrap it in ` i18n._(t``) ` where it is defined.
+- wrap your string using the following format: `` t`String to be translated` ``
 
 **Note:** We try to avoid the `I18n` consumer, `i18nMark` function, or `<Trans>` component lingui gives us access to in this repo. i18nMark does not actually replace the string in the UI (leading to the potential for untranslated bugs), and the other helpers are redundant. Settling on a consistent, single pattern helps us ease the mental overhead of the need to understand the ins and outs of the lingui API.
```
Comment (keithjgrant) [inline awx/ui_next/CONTRIBUTING.md]: While you're in here, we should remove this note about avoiding `<Trans>`

**Review by tiagodread — APPROVED** [review]
All e2e tests are passing, I did an exploratory session manually comparing the results with devel I couldn't find more strings missing i18n.

**Pull request merged**